spec/services/user_anonymizer_spec.rb

```diff
@@ -368,4 +368,17 @@
 
   end
 
+  describe "anonymize_emails" do
+    it "destroys all associated invites" do
+      invite = Fabricate(:invite, email: 'test@example.com')
+      user = invite.redeem
+
+      Jobs.run_immediately!
+      described_class.make_anonymous(user, admin)
+
+      expect(user.email).not_to eq('test@example.com')
+      expect { invite.reload }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
```suggestion
      expect(Invite.exists?(id: invite.id)).to eq(false)
```

Minor point but I think using `.exists?` is more intention revealing. Also, it is slightly faster since the ActiveRecord object does not need to be re-hydrated.